build/cmake/GeneratePkgConfig.cmake

```diff
@@ -282,6 +282,10 @@ function(_generate_pkg_config_content out)
         $<GENEX_EVAL:$<TARGET_PROPERTY:pkg_config_LIBS>>
         $<REMOVE_DUPLICATES:$<TARGET_PROPERTY:INTERFACE_LINK_OPTIONS>>
         )
+
+    # XXX: Could we define a genex that can transform the INTERFACE_LINK_LIBRARIES to a list of
```

Comment:
That's a callout to future readers that may know better than I do, since the current way its handled is sub-optimal but not "broken" to warrant a "TODO" or "FIXME".